readline/readline.go

```diff
@@ -46,8 +60,9 @@ func New(prompt Prompt) (*Instance, error) {
 }
 
 func (i *Instance) Readline() (string, error) {
-	prompt := i.Prompt.Prompt
-	if i.Prompt.UseAlt || i.Pasting {
+	prompt := i.Prompt.prompt()
+	if i.Pasting {
```

Comment:
pasting drops the cursor into a newline which makes this necessary. otherwise it'd be confusing when the prompt shows `>>>` despite being a continuation of the pasted content.

not printing a newline after paste would be ideal. it would remove this edge case and allow the user to add onto the pasted content without forcing a newline

Reply:
I can't replicate this for some reason. Pasting seems the same between the current version and this branch to me. Maybe I dont understand what is being described.

Reply:
In hindsight, this is probably due to me copying the trailing newline, i.e. copying the entire line including the newline